doc/prop.xml

```diff
@@ -8,6 +8,106 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsPartialOrderDigraph">
+<ManSection>
+  <Prop Name="IsPartialOrderDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This is a synonym for <Ref Prop="IsReflexiveDigraph"/> and
+    <Ref Prop="IsAntisymmetricDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>.
+
```

Comment:
Is it `x <= y` if and only if `x -> y` is an edge? This matters for later on when we're talking about upper and lower bounds.